tests/unit/backend/corpora/common/entities/test_project.py

```diff
@@ -114,3 +114,14 @@ def test__create_ids__ok(self):
 
         self.assertEqual(link_ids, get_ids(project.links))
         self.assertNotEqual(["test_project_link_id"], get_ids(project.links))
+
+    def test__list__ok(self):
+        generate = 5
+
+        for _ in range(generate):
+            Project.create(**ProjectParams.get())
+
+        projects = Project.list()
+        self.assertGreaterEqual(len(projects), generate)
```

Comment:
Since the database is not cleared after each tests we don't know exactly how many additional rows have been created.

Reply:
Then we should test exactly which IDs we are expecting. Otherwise, this test can pass without actually testing to make sure that the right projects are returned.